### PR TITLE
Daily fixes

### PR DESCRIPTION
**Pull request by sinkingsugar**
fix fullscreen functionality and drop warning

### CODE REVIEW

**Comment by sinkingsugar** [comment]
I'm making full screen (borderless) default for practical reasons @guusw , have a look please

**Comment by sinkingsugar** [comment]
I think fits also mobile devices where stuff is full screen anyway

**Comment by sinkingsugar** [comment]
Actually I'm not so convinced, maybe we need a fullscreen parameter like before?

File: src/extra/gfx/main_window.cpp [hunk 1]
```diff
@@ -126,6 +138,11 @@ struct MainWindow : public Base {
     WindowCreationOptions windowOptions = {};
     windowOptions.width = _pwidth;
     windowOptions.height = _pheight;
+    if (_pwidth == 0 || _pheight == 0) {
+      windowOptions.fullscreen = true;
+    } else {
+      windowOptions.fullscreen = false;
+    }
```
Comment (Kryptos-FR) [inline src/extra/gfx/main_window.cpp:145]: One issue with this logic is not being able to display in fullscreen at a lower resolution than the screen. Sometimes for performance reason, you want to be fullscreen at `1920*1200` while your screen native resolution is `2560*1440`.

Comment (sinkingsugar) [reply src/extra/gfx/main_window.cpp:145]: In modern rendering you don't do that anymore actually @Kryptos-FR , just use render targets at lower resolution and render all to texture (which you always do due to post effects) 😉

**Comment by guusw** [comment]
I held of on this since having a fixed parameter is not that useful without being able to switch on demand, but I'm fine with adding it like this.

**Review by guusw — APPROVED** [review]

**Comment by sinkingsugar** [comment]
> I held off on this since having a fixed parameter is not that useful without being able to switch on demand, but I'm fine with adding it like this.

But I mean, it's resizable when not fullscreen (windowed) anyway no?

**Comment by sinkingsugar** [comment]
The issue is we make no distinction without maximixed and full screen with this PR

**Comment by sinkingsugar** [comment]
We might want a flag

**Pull request closed without merging**